pkgs/tools/text/uwuify/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, ... }:
+
+let
+  version = "0.1.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "uwu";
```

The package name on crates.io and the project README's title are `uwuify`, so that should prolly be the preferred name.
```suggestion
  pname = "uwuify";
```